visualizer/src/service/service.js

```diff
@@ -0,0 +1,13 @@
+import { interpret } from 'xstate';
+import createDeepcellLabelMachine from './deepcellLabelMachine';
+
+const location = window.location;
+const search = new URLSearchParams(location.search);
+const projectId = search.get('projectId');
+const bucket = search.has('bucket') ? search.has('bucket') : 'caliban-output';
```

Comment:
```suggestion
const bucket = search.has('bucket') ? search.get('bucket') : 'caliban-output';
```
We ran into a small bug where setting an output bucket with the url argument isn't working correctly (although submitting a project to the default bucket works fine). It looks like the output bucket gets set to `true` instead of the specified bucket value.

Reply:
Looks like a good find, but I'll let @tddough98 confirm this and commit in the fix.

Reply:
Fix and redeployed with the fix